lists.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>List component</title>
+    <link rel="stylesheet" href="lists.css"/>
+</head>
+<body>
+    <ul class="lists">
+        <p class="list-title">Menu components</p>
+        <li>Home</li>
+        <li>About</li>
+        <li>
+            <ul class="list-nested">
+                <li>Watches</li>
+                <li>Wallets</li>
+                <li>Windows</li>
+            </ul>
+        </li>
+        <li>Support</li>
+        <li>Contact Us</li>
+    </ul>
```

Comment:
In this list give list-style-type so it will looks good as well easily understandable the list.